selfdrive/car/toyota/interface.py

```diff
@@ -246,7 +246,9 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       set_long_tune(ret.longitudinalTuning, LongTunes.PEDAL)
     elif candidate in TSS2_CAR:
       set_long_tune(ret.longitudinalTuning, LongTunes.TSS2)
-      ret.stoppingDecelRate = 0.3  # reach stopping target smoothly
+      ret.vEgoStopping = 0.2
+      ret.vEgoStarting = 0.2
+      ret.stoppingDecelRate = 0.8  # reach stopping target smoothly
```

Comment:
We can either reach -2.0 m/s/s in the same time stock ACC gets to its standstill accel of -0.4 m/s/s, or use the same rate (take longer to get to -2.0 than stock to -0.4). Not sure if it makes too much of a difference, but this is using the same decel rate stock ACC uses.